### PR TITLE
[Validator] Fix wrong php docs

### PR DESCRIPTION
**Pull request by GuilhemN**
| Q | A |
| --- | --- |
| Fixed tickets | #17091 |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/Validator/Validation.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
      * If you want to configure the validator, use
      * {@link createValidatorBuilder()} instead.
      *
-     * @return ValidatorInterface The new validator.
+     * @return Validator\ValidatorInterface The new validator.
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Validation.php]: please add a `use` statement for the interface instead

**Comment by xabbuh** [comment]
should be merged into `3.0`

File: src/Symfony/Component/Validator/Validation.php (hunk 1 above)
Comment (Ener-Getick) [inline src/Symfony/Component/Validator/Validation.php]: Changed.

**Comment by Ener-Getick** [comment]
I rebased my PR on ``3.0`` is it enough ?

**Comment by Tobion** [comment]
Thanks @Ener-Getick.

**Pull request closed without merging**